src/schema/v1/me/conversations.ts

```diff
@@ -9,6 +9,7 @@ import { ResolverContext } from "types/graphql"
 
 const Conversations: GraphQLFieldConfig<void, ResolverContext> = {
   type: connectionDefinitions({
+    nonNullable: true,
```

Comment:
A bit lower down, on https://github.com/artsy/metaphysics/pull/2346/files?file-filters%5B%5D=.js&file-filters%5B%5D=.patch&file-filters%5B%5D=.ts#diff-0af7e48b8ab9f7dd9d193f44da8f0fc2R24 , there's the `if (!loader) return null` pattern, FYI.

Reply:
Ah ha, I see how I've confused you now :) this `nonNullable` option only applies to the `node` field on an edge, it doesn't affect the `connectionType` returned from `connectionDefinitions`, which will always be the raw nullable version.

I'll update the name of the option to `nodeIsNonNullable`.